encord/dataset.py

```diff
@@ -139,6 +139,10 @@ def add_private_data_to_dataset(
         """
         return self._client.add_private_data_to_dataset(integration_id, private_files, ignore_errors)
 
+    def update_data_item(self, dataset_hash: str, data_item: Video):
+        """This function is documented in :meth:`encord.client.EncordClientDataset.update_data_item"""
```

Comment:
Let's flip this over. We have the pattern to have the documentation here and then have the pointer in our legacy EncordDatasetClient